manifests/q/QtEmu/QtEmu/2.1/QtEmu.QtEmu.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: QtEmu.QtEmu
+PackageVersion: 2.1
+PackageLocale: en-US
+Publisher: QtEmu

```

```suggestion
Publisher: QtEmu Developers
```
This must match Control Panel.